awx/main/analytics/collectors.py

```diff
@@ -33,6 +34,14 @@ def something(since):
 '''
 
 
+def four_hour_slicing(key, since, until):
```

Comment:
I like this 👍

Reply:
I was going to say the same thing - was cool to see this generator 👍